os/windows/env/env.ts

```diff
@@ -15,7 +15,13 @@ type IEnvironmentValueMatch = { groups: { name: string, type: string, data: stri
 
 const REG_KEY = 'HKEY_CURRENT_USER\\Environment'
 
-export async function setupWindowsEnvironmentPath (pnpmHomeDir: string, opts: { force: boolean } = { force: false }): Promise<string> {
+export interface SetupWindowsEnvironmentPathOpts {
```

Comment:
I would make an option to decide if you want it in the end or beginning 
like
`Path=%PNPM_HOME%;C:\foo;C:\bar`
vs
`Path=C:\foo;C:\bar;%PNPM_HOME%`

Reply:
added an option called "position" which may be "start" or "end"